apps/src/templates/rubrics/AiAssessmentFeedback.jsx

```diff
@@ -57,100 +43,43 @@ export default function AiAssessmentFeedback({aiEvalInfo}) {
       Vague: aiVague,
       feedbackOther: aiFeedbackOther,
       otherContent: aiOtherContent,
-    });
-    HttpClient.post(`${baseUrl}`, bodyData, true, {
-      'Content-Type': 'application/json',
-    });
+    };
+
+    submitAiFeedback(bodyData);
 
     setAISubmitted(true);
     setAIFeedbackReceived(true);
-
-    receivedTimer.current = setTimeout(() => {
-      setAIFeedbackReceived(false);
-    }, disappearAfter);
   };
 
   const cancelAiFeedbackCallback = () => {
     //reset all vars
-    setAIFeedback('');
     setAISubmitted(false);
     setAIFalsePos(false);
     setAIFalseNeg(false);
     setAIVague(false);
     setAIFeedbackOther(false);
     setAIOtherContent('');
+
+    // Clear feedback
+    setAiFeedback(-1);
   };
 
   return (
     <div>
-      <div className={style.aiFeedbackRow}>
-        <BodyThreeText>{i18n.aiAssessmentFeedbackAsk()}</BodyThreeText>
-        <label>
-          <span
-            className={
-              aiFeedback === thumbsupval
-                ? style.aiFeedbackThumbsUpChecked
-                : style.aiFeedbackThumbsUp
-            }
-            aria-hidden="true"
-          >
-            {aiFeedback === thumbsupval ? (
-              <FontAwesome icon="thumbs-up" />
-            ) : (
-              <FontAwesome icon="thumbs-o-up" />
-            )}
-          </span>
-          <input
-            type="radio"
-            className={style.aiFeedbackRadio}
-            name={radioGroupName}
-            value={thumbsupval}
-            onChange={() => {
-              radioAiFeedbackCallback(thumbsupval);
-            }}
-            checked={aiFeedback === thumbsupval}
-          />
-        </label>
-
-        <label>
-          <span
-            className={
-              aiFeedback === thumbsdownval
-                ? style.aiFeedbackThumbsDownChecked
-                : style.aiFeedbackThumbsDown
-            }
-            aria-hidden="true"
-          >
-            {aiFeedback === thumbsdownval ? (
-              <FontAwesome icon="thumbs-down" />
-            ) : (
-              <FontAwesome icon="thumbs-o-down" />
-            )}
-          </span>
-          <input
-            type="radio"
-            className={style.aiFeedbackRadio}
-            name={radioGroupName}
-            value={thumbsdownval}
-            onChange={() => {
-              radioAiFeedbackCallback(thumbsdownval);
-            }}
-            checked={aiFeedback === thumbsdownval}
-          />
-        </label>
-        {aiFeedbackReceived && (
-          <EmText className={style.aiFeedbackReceived}>
-            {i18n.aiFeedbackReceived()}
-          </EmText>
-        )}
-      </div>
-
```

Comment:
A lot of this got moved to `AiAssessmentFeedbackRadio` (thumbs up/down buttons) and the submitting happens higher up via the context.